spark/src/main/scala/org/apache/spark/sql/delta/files/TransactionalWrite.scala

```diff
@@ -449,4 +462,27 @@ trait TransactionalWrite extends DeltaLogging { self: OptimisticTransactionImpl
 
     resultFiles.toSeq ++ committer.changeFiles
   }
+
+  /**
+   * Optimized writes can be enabled/disabled through the following order:
```

Comment:
this makes sense, I guess the intention is to have an explicit way to turn on/off for every write/table/session in that order

Reply:
thanks for changing the precendence